libs/core/__deprecated__/renderer/components/rest/menu/menu-bottom.component.tsx

```diff
@@ -71,6 +116,8 @@ const MenuBottom: FunctionComponent<Props> = ({ dataSyncInProgress }) => {
           </Text>
         </SyncProgressWrapper>
       )}
+      {isSelectDevice && SelectDeviceButton}
+      {isChangeDevice && ChangeDeviceButton}
       {!deviceInitialized && devices.length > 0 && (
```

Comment:
please remove this part of the code if you would like to avoid situation like in the bellow screen 👇 
<img width="1637" alt="Zrzut ekranu 2024-01-30 o 08 55 33" src="https://github.com/mudita/mudita-center/assets/17147149/28e200e8-61f8-41da-998e-6d4a80aaa4d6">